Documentation/packaging.md

```diff
@@ -0,0 +1,65 @@
+# packaging rkt
+
+## Build-time dependencies
+
+#### Basic build-time dependencies
+
+- bash
+- go
+- gofmt
+- file
+- git
+- gpg
+
+#### Additional build-time dependencies for the coreos flavor
+
+wget, gpg, mktemp, md5sum, cpio, gzip, unsquashfs, sort
```

Comment:
You might mention that `wget`, `gpg`, `md5sum` and probably `mktemp` are used only when CoreOS image is downloaded from the network.